src/OrleansAWSUtils/Reminders/DynamoDBReminderTable.cs

```diff
@@ -181,6 +214,10 @@ private static ReminderEntry Resolve(Dictionary<string, AttributeValue> item)
             }
         }
 
+        /// <summary>
+        /// Aync method to clear the reminder table
+        /// </summary>
+        /// <returns>A task object </returns>
```

Comment:
There are several occurrences, so I mainly added this on a Task and Task<T> so that it reflects what is important to say here, but it is typically not the fact that the type is a Task, unless the Task completes at a special point in time, and is not necessarily trivial.